File: packages/sdks/javascript/src/services/ProjectService.ts
import { ProjectsApiService } from '../api/generated/apis/projects-api';
import { ProjectsActionsApiService } from '../api/generated/apis/projects-actions-api';
import { ProjectsEnvironmentsApiService } from '../api/generated/apis/projects-environments-api';
import { ProjectsArchivedEnvironmentsApiService } from '../api/generated/apis/projects-archived-environments-api';

export class ProjectService extends ProjectsApiService {
  public actions = new ProjectsActionsApiService();
  public archivedEnvironments = new ProjectsArchivedEnvironmentsApiService();
  public environments = new ProjectsEnvironmentsApiService();
}

File: packages/sdks/python/devopness/services/project_service.py
"""
Devopness API Python SDK - Painless essential DevOps to everyone
"""

from ..generated.api.projects_actions_api import (
    ProjectsActionsApiService,
    ProjectsActionsApiServiceAsync,
)
from ..generated.api.projects_api import (
    ProjectsApiService,
    ProjectsApiServiceAsync,
)

__all__ = ["ProjectService", "ProjectServiceAsync"]


class ProjectService(
    ProjectsApiService,
    ProjectsActionsApiService,
):
    """Service for projects in the Devopness API."""


class ProjectServiceAsync(
    ProjectsApiServiceAsync,
    ProjectsActionsApiServiceAsync,
):
    """Async service for projects in the Devopness API."""
